static/js/brand-store/Settings/Settings.tsx

```diff
@@ -19,30 +19,58 @@ import NotAuthorized from "../NotAuthorized";
 import PasswordToggle from "../shared/PasswordToggle";
 import SectionNav from "../SectionNav";
 
+declare global {
+  interface Window {
+    CSRF_TOKEN: string;
+  }
+}
+
+interface RootState {
```

Comment:
This type appears to be very similar to the one in the test file above...I'm assuming they are the same. What you'll want to do is have a single place where these are typed....typically it would be in this file (or in a separate `types` file). Then the type is exported from that file. You'd then import it using `import type {RootState} from "./Settings"` in the test files (or other files that need it).